src/main/twirl/scalajs/jsdeps.scala.html

```diff
@@ -1,3 +1,5 @@
+@import play.twirl.api.TwirlFeatureImports.defining
```

Comment:
No worries, I've just pushed a commit to upgrade to several libraries (including Twirl `v1.3.0`) https://github.com/vmunier/scalajs-scripts/commit/fab19c75f4b93242354ac87b7a44c7d0086870c0